data/nation/dagestan.ruleset

```diff
@@ -0,0 +1,78 @@
+[nation_dagestan]
+
+name=_("Dagestan")
+plural=_("?plural:Dagestani")
+groups="Asian", "European", "Early Modern"
+
+legend=_("Dagestan is a republic of Russia situated in North Caucausus\
+ along the Caspian sea, it is one of the most ethnically diverse of\
+ Russia's republics, it was one of the first people within the modern\
+ borders of Russia to become Muslims after the Arab conquest of the\
+ region in 643. Dagestan was later raded by the Mongols in 1221.\
```

Comment:
Consider rewriting the first sentence, it's very long and has some inconsistencies ([Dagestan] was one of the first *people* [...] to become Muslims...)